docs-snapshot/declarative.md

```diff
@@ -232,6 +232,8 @@ Same as inbound, the behaviour is defined through overridden method implementati
 *EndpointProducerBuilder defineOutgoingEndpoint()* is used to define the endpoint, 
 which executes the call to an external system.
 StaticEndpointBuilders can be used to provide the endpoint definition. **Apache Camel** is used in the Outbound endpoints as well.
+If the URI of the endpoint contains placeholder values in format of ${placeholder} or {{placeholder}}
```

Comment:
An example like this would be helpful

```
  @Override
  protected EndpointProducerBuilder defineOutgoingEndpoint() {
    return StaticEndpointBuilders.http("https", "api.nobelprize.org/2.0/laureate/${header.id}")
        .bridgeEndpoint(true);
  }
```

Reply:
And setting the value of `${header.id}`
```
  private void setRequest(RouteDefinition routeDefinition) {
    routeDefinition.process(
        exchange -> {
          Integer laureateId = exchange.getMessage().getBody(Integer.class);
          exchange.getMessage().setHeader("id", laureateId);
          // Required by Nobel Prize API, otherwise the response is compressed and cannot be processed
          exchange.getMessage().setHeader("Accept-Encoding", "deflate");
        });
  }
```